apiserver/facades/agent/uniter/access.go

```diff
@@ -78,16 +77,13 @@ func cloudSpecAccessor(authorizer facade.Authorizer, appService ApplicationServi
 			return nil, errors.Errorf("expected names.UnitTag, got %T", tag)
 		}
 
-		appUUID, err := appService.GetApplicationIDByName(ctx, appName)
-		if err != nil {
-			return nil, errors.Trace(err)
-		}
-		config, err := appService.GetApplicationConfig(ctx, appUUID)
```

Comment:
Yeah, that was a whoopsie. We only ever wanted the trust setting for the application, not all the config etc. This is a nice clean up - thanks.